base_custom_attributes/README.rst

```diff
@@ -0,0 +1,28 @@
+Base Custom Attributes 
+===========================================
+This module adds the possibility to easily create custom attributes
+in any OpenERP business object.
+See the product_custom_attributes module for instance.
+
+
```

Comment:
Please add bug tracker link
as in https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
(this is quite a new change you could have missed, it would be nice if you could do it as well)